proxy/api/src/project.rs

```diff
@@ -140,15 +140,6 @@ impl Projects {
 
         Ok(projects)
     }
-
-    /// Give back an `Iter` that can be used to iterate over the projects. It first yields
-    /// contributed projects and then tracked projects.
-    pub fn iter(&self) -> Iter<'_> {
```

Comment:
I mean, it's unused but doesn't mean it's not useful for the future :thinking: How come it doesn't trigger on the `Iter` struct too?

Reply:
This is the same code as for `<Projects as IntoIter>::into_iter()` which probably is used implicitly through a `for .. of`.

Reply:
Ah very good, thanks :grin: